pandas/compat/pickle_compat.py

```diff
@@ -54,6 +55,22 @@ def load_reduce(self):
         raise
 
 
+class _LoadSparseSeries:
```

Comment:
I'm not super familiar with this module but is this definitely something we want to do? Implicitly convert `SparseSeries` from old files to just a `Series`?

Reply:
I think this is something we want to do, to provide some kind of backwards compatibility on older pickle files (note that the output will still be sparse, just not the subclass), and given how easy it is to provide this. 
But, I think we should add a warning to say that the SparseDataFrame has been converted to DataFrame with sparse columns, and to warn this will not be supported anymore in the future (so they need to re-save the files with the new version).

Reply:
Will do.